specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSet_Create_WithDiskControllerType.json

```diff
@@ -0,0 +1,247 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmScaleSetName": "{vmss-name}",
+    "api-version": "2022-08-01",
+    "parameters": {
+      "sku": {
+        "tier": "Standard",
+        "capacity": 3,
+        "name": "Standard_D1_v2"
+      },
+      "location": "westus",
+      "properties": {
+        "overprovision": true,
+        "upgradePolicy": {
+          "mode": "Manual"
+        },
+        "virtualMachineProfile": {
+          "storageProfile": {
```

Comment:
pls. specify the diskcontrollerType property in the request and the same should show up in the response

Reply:
Also, we need to update the examples for requests and responses for Gen2 VMs to add diskcontrollerType property as we will add this property for all Gen2 VMs from api version 2022-08-01.

Reply:
> pls. specify the diskcontrollerType property in the request and the same should show up in the response

Added

Reply:
> Also, we need to update the examples for requests and responses for Gen2 VMs to add diskcontrollerType property as we will add this property for all Gen2 VMs from api version 2022-08-01.

I didnt find any examples specific to Gen 2 VMs.

Reply:
There are examples for Gen2 VMs that shows hyperVGeneration but only for instance view (refer to file VirtualMachine_Get_InstanceView.json), let us add an example that shows the response for GetVM with Instance View where show the response for model (with disk controller type) and instance view (with hyperVGeneration).

Reply:
will add